pages/mesosphere/dcos/services/hive-metastore/1.0.0-3.0.0/release-notes/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes  
+excerpt: Discover the new features, updates, and known limitations in this release of the Hive Metastore Service
+title: Release Notes 
+menuWeight: 10
+model: /mesosphere/dcos/services/hive-metastore/data.yml
+render: mustache
+---
+# Release Notes for the {{ model.techName }} Service version 1.0.0-3.0.0
+
+This is the first release of the {{ model.techName }}! Please read the documentation for details but some highlights include:
+
+- Support for the following databases: Derby, MySQL, Percona-MySQL
+- Being able to easily configure each node's resources
+- Supporting high-availability with the ability to horizontally scale out
+- Automatically generating the `metastore-site.xml` file and exposing it through the `endpoints` CLI command
+- Beeline CLI support
+- Using the SDK version `0.57.0` that enables all the DCOS 2.0 features such as:
+  - node draining
+  - quota enforcement
+  - for more information see the release notes for version [0.57.0](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.0)
+- Support for the [DC/OS Storage Service](https://docs.d2iq.com/mesosphere/dcos/services/storage/)
```

Comment:
I'll add some more release notes if they better serve our purpose to highlight our release features.

Reply:
It might be better to add a section either in `Configuration` or in `Advanced` that describes using volume profiles for DSS

Reply:
I don't know but do we have any template for that? Or should we create a common template that is not specific to a single service and can be used as `include` template in other frameworks, just like `custom-domain` ?

Reply:
Yes it would be great to create a template for DSS If it can be generic. That's what I did for custom-domain in this PR. Nothing is stopping you from doing the same. I just noticed that every other FW was writing the same thing for custom-domain so I opted to make a `tmpl` file

Reply:
I didn't want to update every other FW to use the template since that's outside the scope of this PR but we can either 1) have a separate PR that does just that or 2) have every FW update their docs whenever they update their package

Reply:
I think Point (2) will be better for all, as docs PR are specifically owned and approved by Docs team and it'll require bandwidth from both sides to merge so many changes together. Rather it would be nice to accommodate changes with some releases/hot-fixes where docs changes are specifically required.

Reply:
> We have to remove this bit from the release notes. It turns out we're only on SDK 0.56.1. (Let's just say that instead).

Thanks for clarifying that @samvantran. I'm updating this. This was a point of discussion for yesterday's ping for clarification about the SDK chaos and thus I wanted to be sure about which version we're going GA with.